thirdeye/thirdeye-frontend/app/pods/components/modals/create-event-modal/component.js

```diff
@@ -0,0 +1,50 @@
+/**
+ * Component displaying the event creation inside a modal
+ * @module components/modals/create-event-modal
+ * @property {Function} onSave          - save the events
+ * @property {Function} onExit          - toggle off the modal
+ * @example
+ {{modals/create-event-modal
+   showCreateEventModal=showCreateEventModal
+ }}
+ * @exports create-event-modal
+ * @author jihzhang
+ */
+
+import Component from '@ember/component';
+import {
+  get, set, getProperties
+} from '@ember/object';
+import {inject as service} from '@ember/service';
+
+import fetch from 'fetch';
+
+import createEventApi from 'thirdeye-frontend/utils/api/create-event';
+
+export default Component.extend({
+
+  session: service(), /**
+   * Default selected mapping type
+   * @type {String}
+   */
+  actions: {
+    /**
+     * Handles the close event
+     * @return {undefined}
+     */
+    onExit() {
+      this.set('isShowingModal', false);
+    },
+
+    onSave() {
+      const {
+        startTime, endTime, eventName, countryCode
+      } = getProperties(this, 'startTime', 'endTime', 'eventName', 'countryCode');
+
+      const startTimeSinceEpoch = new Date(startTime).getTime();
```

Comment:
any of the inputs required? If so, we need to confirm and not allow the user to submit.

Reply:
currently not. leave it for now.